### DISCUSSION TITLE
Designer not loading in preview

### DISCUSSION THREAD

**Discussion by SebRHQ** [prplwtf/fetch-nebula#89] [Support] [answered]
![image](https://github.com/user-attachments/assets/fecbfd9b-963f-49e7-b35e-0e6038500a56)

console log:
Refused to frame 'https://29.193.138.95.static.v4yip.hu/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors self".

**Comment by prplwtf** [comment, marked as answer]
This is related to your domain policy, and not Nebula itself. Non-standard domain policies can block Nebula from working correctly, resulting in this issue.
